test/integration/apis/issue.test.ts

```diff
@@ -87,44 +79,101 @@ describe("issue", () => {
         it("should fail if no account is set", () => {
             const issueId: H256 = <H256>{};
             const txId: H256Le = <H256Le>{};
-            const txBlockHeight: u32 = <u32>{};
             const merkleProof: Bytes = <Bytes>{};
             const rawTx: Bytes = <Bytes>{};
-            assert.isRejected(issueAPI.execute(issueId, txId, txBlockHeight, merkleProof, rawTx));
+            assert.isRejected(issueAPI.execute(issueId, txId, merkleProof, rawTx));
         });
 
         it("should request if account is set", async () => {
             issueAPI.setAccount(alice);
             const amount = api.createType("Balance", 1);
             const bobVaultId = api.createType("AccountId", bob.address);
-            const requestResult = await issueAPI.request(amount, bobVaultId);
+            const requestResult = await issueAPI.request(amount);
             txHash = requestResult.hash;
             assert.isDefined(txHash);
```

Comment:
Better would be to check the request issue event for the actual input data. Maybe best would be to expose the helper functions inside the issue api to check for matching events here?